tests/testthat/test_object_to_R_code.R

```diff
@@ -0,0 +1,41 @@
+eval_string <- function(s) eval(str2expression(s))
+
+test_that("Transform vector to R code", {
+  x <- c("A", "B", "C")
+  x_out <- vector_to_R_code(x)
+  expect_equal(eval_string(x_out), x)
+  expect_true(grepl("^c\\(", x_out))
+  expect_true(grepl("\\)$", x_out))
+})
+
+test_that("Transform vector to R code: null handling", {
+  x <- NULL
+
+  x_null <- vector_to_R_code(x)
+  expect_null(eval_string(x_null))
+
+  x_empty <- vector_to_R_code(x, FALSE)
+  expect_equal(eval_string(x_empty), "")
+  expect_true(grepl("^c\\(", x_empty))
+  expect_true(grepl("\\)$", x_empty))
+})
+
+test_that("Express bioavailability as parseable R", {
+  bioav_null <- bioavailability_to_R_code(NULL)
+  expect_equal(eval_string(bioav_null), "1")
+  expect_true(grepl("^c\\(", bioav_null))
+  expect_true(grepl("\\)$", bioav_null))
+
+  vec_char <- c("F1i", "1")
```

Comment:
What about a test for the ternary-style definition, if the plan is to still support that?

Reply:
This change is required to allow for the new specification. It wasn't working without this. The next step is to update all the model files. I locally checked that this would work with one, I can push that up if you'd like to check?

Reply:
The intent is to no longer support ternary-style definitions.

Reply:
OK thx. Yes, can you share one example that isn't working? I tried it for pazopanib and that worked fine.

Reply:
Ah I see, it now only works for vectors, not for character anymore.

Reply:
@roninsightrx here, you can pull this branch: https://github.com/InsightRX/models/pull/380. this PR makes this specification work.